docs/features.md

```diff
@@ -377,7 +377,11 @@ Below you can find the custom linters implemented on clojure-lsp side that uses
 
 For more information on how to configure it, check the [diagnostics settings section](settings.md#diagnostics-linter).
 
-#### clj-depend 
+#### clojure-lsp/uniform-aliasing
+
+For more information on how to configure it, check the [diagnostics settings section](settings.md#diagnostics-linter).
```

Comment:
Would be nice a screenshot or a basic explanation of how this looks like

Reply:
@ericdallo done. Feel free to replace the screenshot with an "emacs" one because VSCode uses a lot of space to show code issues.